Software/frontend/Style/README.md

```diff
@@ -1,17 +1,68 @@
-<<<<<<< Updated upstream
-<!-- press ctrl+K then V to open a preview of the MarkDown file  -->
-=======
->>>>>>> Stashed changes
-# Page Main Title 
-short description of the directory's subject
+# Styling & CSS 
```

Comment:
perhaps this should be a page just for CSS, as it is a broad subject and everything you have added is CSS.
i think styling can be a different page with sites like figma (https://www.figma.com/) and guides to visual design.